app/celer-g4/GeantDiagnostics.cc

```diff
@@ -76,6 +76,9 @@ GeantDiagnostics::GeantDiagnostics(SharedParams const& params)
         output_reg->insert(std::make_shared<BuildOutput>());
     }
 
+    // Create shared exception handler (note: make_shared acts screwy here...)
```

Comment:
```suggestion
```

This was because the accessor name was `MultiExceptionHandler` to follow geant4 conventions, so it shadowed the class name 🤪